modules/addons/ratelimit/redis-image.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+eval "$(jq -r '@sh "TSB_VERSION=\(.tsb_version) REGISTRY=\(.registry)"')"
+
+TMP_DIR=$(mktemp -d)
+helm pull tetrate-tsb-helm/controlplane --version "${TSB_VERSION}" --untar --untardir "${TMP_DIR}"
```

Comment:
it implies that... the end user has a `helm` repository added with the exactly the name... should we instead do `rest api` call towards the `cloudsmith` to retrieve the image? for example i was using this url recently `'[https://api.cloudsmith.io:443/v1/packages/tetrate/getistio-containers/?page=1&page_size=30](https://api.cloudsmith.io/v1/packages/tetrate/getistio-containers/?page=1&page_size=30)'` to get the TID images... I can work towards `TSB` example basically to use latest redis from `https://cloudsmith.io/~tetrate/repos/tsb/packages/?q=redis` output...

another lazy one can be a `curl` of `https://charts.dl.tetrate.io/public/helm/charts/controlplane-1.5.5.tgz` a helm chart? 

the best approach will be to limit the local utilities dependencies or do rest api call within terraform... i.e. helm local setup can be broken, use different names, etc...  or not having a helm at all.